python/graphstorm/config/argument.py

```diff
@@ -1874,9 +1874,128 @@ def lp_edge_weight_for_loss(self):
 
         return None
 
+    @property
+    def train_hard_edge_dstnode_negative(self):
```

Comment:
```suggestion
    def train_hard_negative_edges_dstnode(self):
```

Reply:
I think this is used for custom negative edges instead of only for hard negative edges.

Reply:
Changed to train_etypes_negative_dstnode